velox/functions/prestosql/tests/JsonFunctionsTest.cpp

```diff
@@ -374,12 +374,10 @@ TEST_F(JsonFunctionsTest, jsonParse) {
     velox::test::assertEqualVectors(expected, result);
   }
 
-  // Test try with invalid escape sequence.
```

Comment:
There's still a potential exception that gets thrown at line 239 in velox/functions/prestosql/json/JsonStringUtil.cpp

Is that something Presto would ignore too?  If not could we update these tests to trigger that exception?

Reply:
That error is thrown due to invalid json escape sequences and Presto shows similar behavior. For e.g 
```
SELECT json_parse(x) from (values ('{"\\&page=20": "a", "\\&page=26": "c"}'), ('{"k\\i":"abc","k2":"xyz"}'), ('{"\/&\"\f\r\n": "a", "\/&page=26": "c"}'), ('"abc\u4FE"')) as t(x);
``` 

I modified the test case to conform to this behavior.